libbeat/processors/condition.go

```diff
@@ -120,10 +121,17 @@ func (c *Condition) setEquals(cfg *ConditionFields) error {
 			c.equals[field] = EqualsValue{Int: uintValue}
```

Comment:
Perhaps we should use `continue` in the code to have less nested if / else? Same on line 125.